src/service/ui/src/controllers/user.ts

```diff
@@ -120,9 +136,9 @@ export function updateProfile(req: Request, res: Response) {
 		uri: config.AUTHENTICATION.serviceUrl + '/scim2/Users/' + req.user.id,
 	}
 
-	request(options, (error, response, body) => {
-		if (!error && response.statusCode === 200) {
-			updateUserObject(req, updateProfileObject.CshrUser)
+	request(options, (error: Error, response: Response, body: Body) => {
```

Comment:
Any reason for adding these type annotations here? I think the `Response` we have here is `express.Response` which isn't what `request` is expecting?